client/client-multi/client-c/src/kaa_profile.c

```diff
@@ -374,6 +390,7 @@ kaa_error_t kaa_profile_handle_server_sync(kaa_profile_manager_t *self
 
 kaa_error_t kaa_profile_manager_update_profile(kaa_profile_manager_t *self, kaa_profile_t * profile_body)
```

Comment:
I know it isn't your fault, but could you please fix the code style of the pointer usage in the parameter list of each function? :)